### PR TITLE
BP-838  Add logo's for inline (custom)giftcards.

### PR DESCRIPTION
**Pull request by saranchuk-hys**

### CODE REVIEW

File: Block/Adminhtml/Grid/Renderer/Image.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        parent::__construct($context, $data);
+        $this->_authorization = $context->getAuthorization();
+    }
+
+    public function render(DataObject $row)
+    {
+        if ($img = $row['logo']) {
+            $mediaDirectory = $this->storeManager->getStore()->getBaseUrl(
+                \Magento\Framework\UrlInterface::URL_TYPE_MEDIA);
+            return '<img src="' . $mediaDirectory . $img . '" width="50"/>';
```
Comment (vladislav-padalka-hys) [inline Block/Adminhtml/Grid/Renderer/Image.php:44]: no trailing slash is needed in html5, especially without space between double quote

File: Controller/Adminhtml/Giftcard/Save.php [hunk 2]
```diff
@@ -20,21 +20,55 @@
...
-            $giftcardId = $this->getRequest()->getParam('entity_id');
+            $giftcardId    = $this->getRequest()->getParam('entity_id');
 
             if ($giftcardId) {
                 $giftcardModel->load($giftcardId);
             }
 
             $formData = $this->getRequest()->getParam('giftcard');
+
+            if ((isset($_FILES['logo']['name'])) && ($_FILES['logo']['name'] != '') && (!isset($formData['logo']['delete']))) {
```
Comment (vladislav-padalka-hys) [inline Controller/Adminhtml/Giftcard/Save.php:42]: $this->getRequest()->getFiles()
will it work in more magento native way ?

File: Block/Adminhtml/Grid/Renderer/Image.php (hunk 1 above)
Comment (saranchuk-hys) [reply Block/Adminhtml/Grid/Renderer/Image.php]: fixed

File: Controller/Adminhtml/Giftcard/Save.php (hunk 2 above)
Comment (saranchuk-hys) [reply Controller/Adminhtml/Giftcard/Save.php]: fixed

**Review by vladislav-padalka-hys — APPROVED** [review]

**Pull request merged**